feathr_project/feathr/registry/feature_registry.py

```diff
@@ -55,9 +55,14 @@ def save_to_feature_config_from_context(self, anchor_list, derived_feature_list,
 def default_registry_client(project_name: str, config_path:str = "./feathr_config.yaml", project_registry_tag: Dict[str, str]=None, credential = None) -> FeathrRegistry:
     from feathr.registry._feathr_registry_client import _FeatureRegistry
     from feathr.registry._feature_registry_purview import _PurviewRegistry
+    from feathr.registry._feathr_registry_client_aws import _FeatureRegistryAWS
+    from aws_requests_auth.aws_auth import AWSRequestsAuth
```

Comment:
Sounds like this part is irrelevant for this PR, maybe we have a seperate PR for this?

Reply:
great point! I have removed the registry changes from this PR